projects/1.18-fabric/assets/auto-workstations-fabric/autoworkstations/lang/zh_cn.json

```
@@ -0,0 +1,37 @@
+{
+  "itemGroup.autoworkstations.items": "自动化工作站",
+  "block.autoworkstations.iron_auto_crafting_table": "铁制自动工作台",
+  "block.autoworkstations.iron_auto_crafting_table.tooltip": "自动合成",
+  "block.autoworkstations.gold_auto_crafting_table": "金制自动工作台",
+  "block.autoworkstations.gold_auto_crafting_table.tooltip": "一个速度更快的自动工作台",
+  "block.autoworkstations.netherite_auto_crafting_table": "下界合金制自动工作台",
+  "block.autoworkstations.netherite_auto_crafting_table.tooltip": "可以一次性合成多达一组的物品",
+  "block.autoworkstations.iron_auto_furnace": "铁制自动熔炉",
+  "block.autoworkstations.iron_auto_furnace.tooltip": "可以将熔炼所获得的经验装瓶",
+  "block.autoworkstations.gold_auto_furnace": "金制自动熔炉",
+  "block.autoworkstations.gold_auto_furnace.tooltip": "一个速度更快的自动熔炉",
+  "block.autoworkstations.auto_enchanting_table_xp_inside": "不消耗经验的自动附魔台",
+  "block.autoworkstations.auto_enchanting_table_xp_inside.tooltip": "假装这个物品不存在",
+  "block.autoworkstations.iron_auto_enchanting_table": "铁制自动附魔台",
+  "block.autoworkstations.iron_auto_enchanting_table.tooltip": "用附魔之瓶来附魔物品，摆放书架以提升魔咒等级",

```

```suggestion
  "block.autoworkstations.iron_auto_enchanting_table.tooltip": "用附魔之瓶来附魔物品，摆放书架以提升附魔等级",
```
根据语境，应该指的是附魔台三个选项右下角的数字
为之前的错误审核致歉